src/LndClient/Data/OpenChannel.hs

```diff
@@ -26,10 +26,10 @@ instance ToGrpc OpenChannelRequest GRPC.OpenChannelRequest where
   toGrpc x =
     msg
       <$> toGrpc (nodePubkey x)
-      <*> toGrpc (localFundingAmount x)
-      <*> toGrpc (pushSat x)
+      <*> (toGrpc =<< toSat (localFundingAmount x))
```

Comment:
here, as we have MSat and need to transform it to Sat, which is returning Either (the same as toGrpc) I've used bind 
`toGrpc =<< toSat`